clang/lib/Headers/CMakeLists.txt

```diff
@@ -501,6 +501,10 @@ add_header_target("windows-resource-headers" ${windows_only_files})
 add_header_target("utility-resource-headers" ${utility_files})
 
 get_clang_resource_dir(header_install_dir SUBDIR include)
+target_include_directories(clang-resource-headers INTERFACE
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
```

Comment:
Do we need to worry about other files accidentally being made visible by using the source directory?

Reply:
Good point. I think I was trying to see if I could remove one set of header copies, but didn't actually remove that bit. Let me point it at the intermediate directory in the build directory instead which should only contain the headers for the targets we've configured LLVM for.